lib/chef/provider/package/chocolatey.rb

```diff
@@ -130,6 +130,17 @@ def remove_package(names, versions)
         # install from, but like the rubygem provider's sources which are more like repos.
         def check_resource_semantics!; end
 
+        def get_choco_version
+          @get_choco_version ||= powershell_exec!("choco --version").result
```

Comment:
I'm getting:

```
Unexpected exit in PowerShell command: ["choco: The term 'choco' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.\nAt line:1 char:1\r\n+ choco --version\r\n+ ~~~~~"]
```

Should this have been:

```suggestion
          @get_choco_version ||= powershell_exec!("#{choco_exe} --version").result
```

Reply:
Oh, that doesn't work because `choco_exe` is only available in the instance.